server/chat-commands/admin.ts

```diff
@@ -1122,17 +1122,42 @@ export const commands: Chat.ChatCommands = {
 		this.stafflog(`${user.name} used /endemergency.`);
 	},
 
-	kill(target, room, user) {
+	async savebattles(target, room, user) {
+		this.checkCan('rangeban'); // admins can restart, so they should be able to do this if needed
+		this.sendReply(`Saving battles...`);
+		const count = await Rooms.global.saveBattles();
+		this.sendReply(`DONE.`);
+		this.sendReply(`${count} battles saved.`);
+		this.addModAction(`${user.name} used /savebattles`);
+	},
+
+	async kill(target, room, user) {
 		this.checkCan('lockdown');
+		const noSave = toID(target) === 'nosave';
 
-		if (Rooms.global.lockdown !== true) {
-			return this.errorReply("For safety reasons, /kill can only be used during lockdown.");
+		if (Rooms.global.lockdown !== true && noSave) {
```

Comment:
Shouldn't this also check if battles _can_ be saved and if they cannot, set noSave to true? This will allow `/kill`ing without lockdown or saving if postgresql is not installed or configured.

Reply:
Yeah. Fixed.